core/pom.xml

```diff
@@ -573,6 +573,11 @@
 			<version>2.3.5</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.5</version>
+		</dependency>
```

Comment:
Why do we need this dependency? It's never used?

Reply:
I used it for conversion to JSON, but it's not needed since I now use the already org.json dependency.